todo_app/app.py

```diff
@@ -1,11 +1,18 @@
-from flask import Flask
-
+from flask import Flask, redirect, url_for, render_template, request
+from todo_app.data.session_items import get_items,add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route('/index')
```

Comment:
Currently it gives an error when I go to the base URL, which is weird.

You will want to have something at '/'. You could make  '/' to redirect to '/index', but having it as '/' makes the index URL clean.

Reply:
amended